packages/pwa-kit-runtime/README.md

```diff
@@ -19,3 +19,7 @@ The full documentation for PWA Kit and Managed Runtime is hosted on the [Salesfo
 -   [Routing](https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/routing.html)
 -   [Phased Headless Rollouts](https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/phased-headless-rollouts.html)
 -   [Launch Your Storefront](https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/launching-your-storefront.html)
+
+## Support Policy
+Customers on existing support contracts are entitled to bug fixes and updates to the latest version of each major SDK version,
+for a period of 24 months from the time of a given version’s General Availability release.
```

Comment:
```suggestion
## Update Policy
Bug fixes, security patches, and other updates are provided for 24 months after the general availability of each major version of the SDK (1.0, 2.0, and so on).
```

Reply:
We're getting close! We are **only** committing to provide security patches, not bug fixes or other updates in this policy.